internal/dev_server/api/server.go

```diff
@@ -50,11 +50,18 @@ func (s Server) PostDevProjectsProjectKey(ctx context.Context, request PostDevPr
 }
 
 func (s Server) DeleteDevProjectsProjectKeyOverridesFlagKey(ctx context.Context, request DeleteDevProjectsProjectKeyOverridesFlagKeyRequestObject) (DeleteDevProjectsProjectKeyOverridesFlagKeyResponseObject, error) {
-	//TODO implement me
-	panic("implement me")
+	store := model.StoreFromContext(ctx)
+	err := store.DeleteOverride(ctx, request.ProjectKey, request.FlagKey)
+	return DeleteDevProjectsProjectKeyOverridesFlagKey200JSONResponse{}, err
```

Comment:
not filling out this response body - it'll just default to false/null anyway which feels ok?

Reply:
Maybe a 204 would be better if you're expecting an empty response?

Reply:
We should either 204 or keep it 200 & return like `{Override: false, Value:...}`.

Reply:
updated to 204